### ISSUE TITLE
`README.md` npm example's module usage is confusing

### ISSUE THREAD

**Issue by akihikodaki** [pixiv/three-vrm#144]
The module usage of the "npm" example in `README.md` is confusing. The example has the following `import` statements:
```JavaScript
import * as THREE from 'three';
import { GLTFLoader } from 'three/examples/jsm/loaders/GLTFLoader';
import { VRM } from '@pixiv/three-vrm';
```

However, `GLTFLoader` and `VRM` is apparently referred as `THREE.GLTFLoader` and `THREE.VRM`. Then what does the literal `GLTFLoader` and `VRM` does?

Also, it does not tell how `import` should be interpreted. It matters because there are number of options for interpretation (e.g. the "babel" transpiler, webpack, and Node.js's ESM.)

**Comment by rdrgn** [comment]
I made a pull request to fix `THREE.GLTFLoader` and `THREE.VRM` (#143).
The other issue may take a bit more discussion with authors.

**Comment by FMS-Cat** [comment]
yes, as the @rdrgn 's PR says, the previous example code was very wrong, I'm so sorry about it!

**Comment by FMS-Cat** [comment]
I will finally mark this as resolved after we merge #527 :bow: :bow: :bow: :bow: :bow:
Going to explicitly state that `.js` builds are umd and `.module.js` builds are esm.
